__tests__/app.test.js

```diff
@@ -84,3 +85,33 @@ describe("GET article by its id", () => {
       });
   });
 });
+describe("GET api/articles", () => {
+  test("GET status 200- should respond with the an articles array of objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        const { articles } = response.body;
+        articles.forEach((article) => {
```

Comment:
You should check the length of the array if asserting inside a forEach, please add this after your forEach on line 96 👍